src/cmd/vendored.go

```diff
@@ -71,6 +73,40 @@ var zarfCmd = &cobra.Command{
 	DisableFlagParsing: true,
 }
 
+// uds-security-hub CLI command
+var scanCmd = &cobra.Command{
+	Use:   "scan",
+	Short: "[ALPHA] Scan a zarf package for vulnerabilities and generate a report.",
+	Long:  "[ALPHA] Scan a zarf package for vulnerabilities and generate a report.",
+	Run: func(cmd *cobra.Command, _ []string) {
+		org, _ := cmd.Flags().GetString("org")
```

Comment:
Instead of rewriting the cmd flags, what do you think of doing something like this:

```
// uds-security-hub CLI command
var scanCmd = &cobra.Command{
	Use:   "scan",
	Short: "[ALPHA] Scan a zarf package for vulnerabilities and generate a report.",
	Long:  "[ALPHA] Scan a zarf package for vulnerabilities and generate a report.",
	Run: func(cmd *cobra.Command, _ []string) {
		os.Args = os.Args[1:] // grab 'scan' and onward from the CLI args
		securityHub.Execute(os.Args)

	},
	DisableFlagParsing: true,
}
```

Reply:
I have taken care of it. Thanks